src/tests/grid/test_stretchedrotatedgaussian.cc

```diff
@@ -0,0 +1,515 @@
+#include "atlas/library/Library.h"
+#include "atlas/util/Config.h"
+#include "atlas/grid.h"
+#include "atlas/array.h"
+#include "atlas/parallel/mpi/mpi.h"
+#include "atlas/functionspace.h"
+#include "atlas/util/UnitSphere.h"
+#include "tests/AtlasTestEnvironment.h"
+
+#include <algorithm>
+#include <vector>
+
+#include <sys/time.h>
+#include <stdio.h>
+#include <math.h>
```

Comment:
- `sys/time.h` is a C header which seems unused (?)
- `stdio.h` also seems unused (as printf is commented)
- Instead of `math.h` please use <cmath> and replace `fabs(...)` with `std::abs(...)` (see below)

Reply:
Also, the usual ordering is system, then standard headers, then external dependencies headers, and finally project headers, all sorted alphabetically if possible.